DotBot.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, dotBotId, x, y, floorplan):
         # estimated PDR is number of packets received over number of packets expected
         self.estimatedPdr         = 1      # packets received per second
         # how frequent estimated PDR is sent to orchestrator
-        self.estimatedPdrPeriod   = 10     # in seconds
+        self.estimatedPdrPeriod   = 5     # in seconds
```

Comment:
why the change? I don't disagree, just trying to understand the reasoning to speed things up.

Reply:
because with 10 seconds the robots would sometimes lose connectivity before the orchestrator receives a new PDR estimation as there is a large delay between actual PDR and the estimated value the orchestrator has